src/MICore/CommandFactories/lldb.cs

```diff
@@ -160,5 +160,12 @@ public override Task Catch(string name, bool onlyOnce = false, ResultClass resul
         {
             throw new NotImplementedException("lldb catch command");
         }
+
+        public async override Task<string> VarAssign(string variableName, string expression, int threadId, uint frameLevel)
+        {
+            Results results = await VarCreate(expression, threadId, frameLevel, 0, ResultClass.done);
```

Comment:
The argument being passed as `0` is the `enum_EVALFLAGS` passed to evaluate.

I see it's not actually being used. Should I instead construct some other flags more descriptive of the type of evaluations that we want to allow here (even if it's not enforced), or rather take this as a paramter for `VarAssign` and pass some flags from the client of this method (in `Variables.cs`).